neo/Consensus/Helper.cs

```diff
@@ -26,8 +27,16 @@ internal static class Helper
         public static bool CommitSent(this IConsensusContext context) => context.CommitPayloads[context.MyIndex] != null;
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static bool BlockSent(this IConsensusContext context) => context.Block != null;
+
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool ViewChanging(this IConsensusContext context) => context.ChangeViewPayloads[context.MyIndex]?.GetDeserializedMessage<ChangeView>().NewViewNumber > context.ViewNumber;
+        public static bool ViewChanging(this IConsensusContext context) => context.ChangeViewPayloads[context.MyIndex]?.GetDeserializedMessage<ChangeView>().NewViewNumber > context.ViewNumber && !context.MoreThanFNodesCommitted();
```

Comment:
This changed the original meaning of `ViewChanging`.

Reply:
Yes it did, because it made the code changes less and fairly clear. I can adjust it to use a separate flag and add more code if you like.

Reply:
I agree with you, @erikzhang.
The original meaning was discussed today with @igormcoelho and it means to not accept anything after change view.

But this is the edge case in which 2 nodes commited and 2 are trying to change view (in a network with 4 nodes).
This is currently hapenning and stucking the network very often in our local experiments.

I think this do not have negative impacts and is a good exception.

Reply:
We can change the name of the `ViewChanging` flag, what do you think @erikzhang?

Reply:
`ViewChangingIfAllowed`?

Reply:
@erikzhang, if you really do not like it we could remove it. Your arguments are reasonable because they are based on proofs from the literature.
But our local privatenet will surely keep being stucked.

This is like an idea that can increase the overall experience.

Reply:
I like the name changing, @jsolman.

Reply:
I think we should remove this and see...

My tests are in the extreme cases.
This case maybe can be removed without loss of power of the dBFT.

Reply:
We cannot remove this case.

Reply:
@erikzhang I think I got what they meant... We could preserve flag `ViewChanging()` as it was before, with the original meaning, and create another one `NotAcceptingPayloads()`, that has this new behavior.
So, on parts which we depend on ViewChanging, we keep that, and other lines we use `NotAcceptingPayloads()`
I think this resolves the issue.

Reply:
@igormcoelho Thanks, I renamed the flag similar to as you suggested. I'm going to resolve this one now, and we can open another if we don't like the new name I gave.